### PR TITLE
Add more configurable log level

### LINKED ISSUE #2184: Configurable logging
# What is needed?
Every time that Debug is enabled, is enabled in all the code and is hard to follow sometimes. 
With this enhancement we should be able to enable `DEBUG` for some parts of code, as for example to everything related with ERC20 indexer, and leave the rest in `INFO` level. 
# Log groups proposal

    ERC20_721_INDEXER_LOG_LEVEL
    PROXY_FACTORY_INDEXER_LOG_LEVEL
    SAFE_EVENTS_INDEXER_LOG_LEVEL
    INTERNAL_TX_INDEXER_LOG_LEVEL
    API_LOG_LEVEL (all the endpoints)
    BALANCES_API_LOG_LEVEL
    MESSAGES_API_LOG_LEVEL
    ALL_TRANSACTIONS_API_LOG_LEVEL

# Proposed solution
Django logging is configurable and we can define different environment variables to enable just DEBUG in some parts of the service. For example:
```
#ERC20_LOGGER
       "safe_transaction_service.history.indexers.erc20_events_indexer": {
            "level": env("ERC20_INDEXER_LEVEL", default="INFO")
        },
        "safe_transaction_service.history.tasks.index_erc20_events_task": {
            "level": env("ERC20_INDEXER_LEVEL", default="INFO")
        },
```
Just remark that if `DEBUG` is enabled in general way all the parts of the code should be log level `DEBUG`

### PR DESCRIPTION
**Pull request by moisses89**
Closes  #2184

### CODE REVIEW

**Comment by moisses89** [comment]
@Uxio0 maybe we should add also a configuration to EventsIndexer and EthereumClient, but the issue with this classes are that the log level will be inherited byt the child classes as SafeEventsIndexer and ProxyFactoryIndexer.

**Review by falvaradorodriguez — APPROVED** [review]

**Review by falvaradorodriguez — APPROVED** [review]
